src/bin/loader.rs

```diff
@@ -0,0 +1,19 @@
+use std::time::{Duration, Instant};
```

Comment:
do we need this function? is it the same as `stress`?

Reply:
Yes but with this we don't need an external binary

Reply:
Replaced with a simple bash script